src/mame/mr/dribling.cpp

```diff
@@ -453,9 +501,33 @@ void dribling_state::dribling(machine_config &config)
 	PALETTE(config, "palette", FUNC(dribling_state::palette), 256);
 
 	// sound hardware
-}
-
+	SPEAKER(config, "mono").front_center();
+
+	NETLIST_SOUND(config, "snd_nl", 12500)
+		.set_source(NETLIST_NAME(dribling))
+		.add_route(ALL_OUTPUTS, "mono", 1.0);
+
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(0),  "I_PB0.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(1),  "I_PB1.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(2),  "I_PB2.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(3),  "I_PB3.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(4),  "I_PB4.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(5),  "I_PB5.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(6),  "I_PB6.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_i_pb.at(7),  "I_PB7.IN", 0);
```

Comment:
You’re supposed to index array-like things like arrays – `m_i.pp[0]` etc.  Using `at(...)` has additional overhead because it does a bounds check to throw an exception if you pass it a bad index, which you can tell won’t happen at compile time.